src/components/DroughtHistory.vue

```diff
@@ -105,18 +102,11 @@ export default {
 
     // Define scrollTo dates
     this.scrollToDates = [
-      {id: '1920s', date: '1921-10-01'}, 
-      {id: '1930s', date: '1930-01-01'}, 
-      {id: '1940s', date: '1940-01-01'}, 
-      {id: '1950s', date: '1950-01-01'}, 
-      {id: '1960s', date: '1960-01-01'},
-      {id: '1970s', date: '1970-01-01'},
-      {id: '1980s', date: '1980-01-01'},
-      {id: '1990s', date: '1990-01-01'},
-      {id: '2000s', date: '2000-01-01'},
-      {id: '2010s', date: '2010-01-01'}]
-    // Get all possible button ids
-    this.decadeIDs = this.scrollToDates.map(scrollDate => scrollDate.id);
+      {id: '1930', name: 'Dust Bowl', start: '1930-02-01', end: '1941-08-31'}, 
+      {id: '1952', name: '1950s Drought', start: '1952-11-01', end: '1957-08-31'}, 
+      {id: '1962', name: '1960s Drought', start: '1962-12-01', end: '1968-10-31'}, 
```

Comment:
As a test case, when I press the 1960s drought button, it takes me to this view:

![image](https://github.com/USGS-VIZLAB/drought-timeline/assets/19958841/4ff2f390-0997-4b9d-b8ca-0b321e5f9bb5)

The drought began in December 1962, but the 1963 line isn't showing here. I'm thinking it's because the buttons are obscuring a bit of the png?

Reply:
Yeah, I was noticing that, too - let me add in an offset, so that the scroll-to rectangles actually start just before the droughts

Reply:
New scroll stop is offset:
![image](https://github.com/USGS-VIZLAB/drought-timeline/assets/54007288/6ce0e0e9-5123-4bf4-b9ea-3e68514dbff0)

on mobile:
![image](https://github.com/USGS-VIZLAB/drought-timeline/assets/54007288/b5bbebaf-3224-4ad3-a760-1a64aa9ef68b)

Reply:
Looks good on Firefox mobile and Safari desktop!

![image](https://github.com/USGS-VIZLAB/drought-timeline/assets/19958841/60f5c8c4-ed98-4d13-a24a-7bb229277527)